### ISSUE TITLE
Native support of SVG inside XAML for Responsive UI and Hardware acceleration backend on OpenGL/Direct3D/Vulkan/Metal

### ISSUE THREAD

**Issue by insinfo** [Microsoft/xaml-standard#112]
Native support of SVG inside XAML for Responsive UI and Hardware acceleration backend on OpenGL/Direct3D/Vulkan/Metal

**Comment by mossyblog** [comment]
I'm dubious around platform injection parameters but agree on spirit svg "as-is" can and should be accepted .. however he concern lies in whether it's inline or path pointer focused as the nature of svg can be quite verbose 

Having import svg can alleviate pressure on design tooling pipelines to get involved with the standard as they already support this today

**Comment by insinfo** [comment]
@mossyblog
I think the important thing is that SVG is supported in a way that is easy to create buttons and other controls with embedded SVG including inline SVG animations

**Comment by mossyblog** [comment]
@insinfo agreed provided its not platform specific was my area of thought.. like thats a really slippery slope in predicate logic conditions... especially how carefree mobile platforms can be.. yes i'm looking at you iOS ...

**Comment by birbilis** [comment]
there are at least two other issues mentioning SVG

**Comment by insinfo** [comment]
I updated with a more coherent syntax

**Comment by michael-hawker** [comment]
I think supporting vector animations is really important, but not sure if the issue should call out a specific technical solution in the title.  SMIL seems to be being replaced on the web (https://developer.mozilla.org/en-US/docs/Web/SVG/SVG_animation_with_SMIL,
https://wpdev.uservoice.com/forums/257854-microsoft-edge-developer/suggestions/6509024-svg-animation-elements).  While it seemed convenient, I feel like it'd be hard to take on and split web/desktop pipelines.  

Think this needs to be thought of with Issue #106.

**Comment by insinfo** [comment]
In fact, google chrome is the only one that has marked SMIL as deprecated, but will still support it for a while, since there is not yet a standard that is complete like SMIL, WEB Animation is still in development and has not been finalized by W3C and CSS Animations is not as complete as SMIL.

SMIL is maintained and updated by W3C and there is no direct replacement for it.

**Comment by insinfo** [comment]
"WEB Animation" and "CSS Animations" was developed for another purpose

**Comment by michael-hawker** [comment]
I'm not saying we should necessarily align to web standards, but that we should make sure we're not depending on another standard which is in decline (e.g. SMIL is planned to be removed from Chrome, with the largest browser share, and not natively supported in IE or Edge).  

Similarly though, we *should* make sure whatever animation technology is used for XAML Standard is compatible with existing design pipelines.  Maybe that means some implementations of XAML can support SMIL for now, but I think there's a general animation discussion of what the *main root system* is that is being discussed in #106.
